2025/cfp-main-2025.md

```diff
@@ -0,0 +1,184 @@
+   5. Audio/video facilities to support speakers and talk recordings.
+8. The on-site team must provide a local expenses budget plan as part of the proposal. An example budget plan can be provided on request. The on-site team will provide details of how they will fund the conference in support of the expenses expected. As far as possible, conference fees should be kept to a minimum to support the participation of attendees from the Asia Pacific / Australasia region. In support of the potential PyCon APAC host, requests for financial support can be made to the PAC to cover specific items of the budget plan. **The finances associated with the running of the conference must be maintained in a manner that ensures transparency and proper financial governance.**
+9. The on-site team must be able to provide supporting letters for visa applicants wanting to attend the conference.
+
+# Proposal Structure<a name="structure"></a>
+
+The proposal must cover all of the following points:
+
+1. A proposal introduction that describes the motivation for the proposal.
+2. A description of the on-site team, its members, and its history in the context of their local Python community.
+3. Short biography of the key members and their previous experience in conference and events organizations.
+4. Previous conference history of the on-site team (if any). Describe which conferences were run and provide some figures on their size (registrations, tracks, total income).
+5. Assignment of key positions to on-site team members. Positions to be assigned (one member can be responsible for more than one of the following roles):
+   1. Chairperson of the on-site team workgroup: usually also the conference chair and representative of the conference for that year.
+   2. Sponsor manager: on-site contact for all sponsor activities, responsible for local sponsors.
+   3. Venue manager: handling all contacts with the venue and caterer.
+   4. Logistics manager: handling all aspects of local logistics and customs.
+   5. Conference desk manager: responsible for the on-site help desk, on-site registrations, tickets, cash refunds, attendee support, etc.
+6. List of all team members that will be present during the conference itself as volunteers (but not necessarily help before the conference begins), and/or sources from which people can be or will be acquired if needed (e.g., universities, other local associations, etc.). This list is indicative, as most of the volunteers helping during the conference are usually enrolled during the last months/weeks before the conference.
+7. List of potential international and local sponsors. Note: Local sponsors help a lot in funding the conference. These are often smaller companies that will not move to new locations with the conference, so a local contact is highly beneficial in attracting these sponsors.
+   1. The PAC can assist with contacts with international sponsors where possible.
```

Comment:
`PAC` ??

Reply:
Thanks for pointing that out. I have added a definition for PAC at the start.

Now, since PAO hasn't officially started, and we plan to announce it during PyCon APAC this year, this CfP will still be under the "old" community guidelines. For next year, we will migrate this document and use PAO instead.